consumer-services/src/main/scala/za/co/absa/spline/consumer/service/model/AccessValue.scala

```diff
@@ -0,0 +1,9 @@
+package za.co.absa.spline.consumer.service.model
+
+object AccessValue extends Enumeration {
+  type AccessValue = Value
+
+  val read = Value("read")
```

Comment:
Please use capitalized names for constant values (`Read` instead of `read`) according to [Scala naming convention](https://docs.scala-lang.org/style/naming-conventions.html#constants-values-variable-and-methods)

Reply:
Also please use more descriptive name for enumeration type. E.g. `AccessType`? Or maybe `DataSourceActionType`?

Reply:
Added